pkg/controller/storagecluster/update.go

```diff
@@ -704,6 +702,8 @@ func matchSelectedFields(
 		return false, nil
 	} else if !reflect.DeepEqual(oldSpec.Resources, currentSpec.Resources) {
 		return false, nil
+	} else if !reflect.DeepEqual(oldSpec.CSI, currentSpec.CSI) {
```

Comment:
Let's remove this now as we are already checking if the portworx pod needs to be bounced below based on the CSI spec.
Also, this will cause the portworx pod to bounce even if some one enables the snapshot controller.

Reply:
good catch, left this in by accident - was originally testing here before adding isCSIBounceRequired...